python/cdk_constructs/api.py

```diff
@@ -44,36 +42,28 @@ def __init__(
         rest_api = aws_apigateway.RestApi(
             scope=self,
             id=f"{id}RestApi",
-            default_method_options=aws_apigateway.MethodOptions(
-                authorization_type=aws_apigateway.AuthorizationType.COGNITO,
-                authorizer=aws_apigateway.CognitoUserPoolsAuthorizer(
-                    scope=self,
-                    id=f"{id}Authorizer",
-                    cognito_user_pools=[user_pool],
-                ),
-            ),
         )
 
         # Api Resources
         badges_resource = rest_api.root.add_resource("badges")
-        # available_badges_resource = rest_api.root.add_resource("avialable_badges")
-        # auth_resource = rest_api.root.add_resource("auth")
+        available_badges_resource = rest_api.root.add_resource("available_badges")
```

Comment:
Not sure I follow what would the difference be of `badges` and on line 49 `available_badges`.

Reply:
well actually it will be useful I think. In front end i can let a user toggle a button if they want to see `available_badges` or all the `badges` so they can see what their options are. Makes sense

Reply:
Will rename according to what we discussed on the phone today - `all_badges` with the ability to filter on status.